extensions/agroal/runtime/src/main/java/io/quarkus/agroal/DataSource.java

```diff
@@ -21,5 +21,5 @@
 @Qualifier
 public @interface DataSource {
 
-    String value() default "";
```

Comment:
Why did we take out the default?

Reply:
Because it's counter productive? The value has to be defined as it is used for named datasources only. Or did I miss something?

Reply:
Okay, yeah, I guess that works. You only use it for named datasources, so it makes sense.